resources/metadata/cloudFoundryDeploy.yaml

```diff
@@ -370,6 +370,11 @@ spec:
   containers:
     - name: cfDeploy
       image: ppiper/cf-cli:latest
+      options:
+        - name: --ulimit
+          value: "stack=67108864:67108864"  # Stack Size (in bytes). 64mb
+        - name: --ulimit
+          value: "nofile=65536:65536"       # Number of Open Files
```

Comment:
How those values were calculated? Was performance tested with these values? Do we need to set these values for all the builds or it could be an additional parameter for rare cases?

Reply:
User tested changes and confirmed the issue is solved for them. Build time not changed. No any configuration change need on orchestrator size.

Setting ulimit to these values is common practice to avoid "argument list too long" issue on Unix-based systems, so that docker provides option for this (https://docs.docker.com/reference/cli/docker/container/run/#ulimit) which I used. I provided some additional links in the related SNOW ticket.

By default stack limit is 8mb (for both command line+arguments and environment variables) and we increased it to 64mb here (64*1024*1024=67,108,864 bytes).

New increased limits applied to all pipelines that using cloudFoundryStep (as it used same docker container with the parameters defined here)